[ISSUE] Add description to selectTree #664 (#740)
TYPO3-Documentation/TYPO3CMS-Reference-TCA: Documentation/ColumnsConfig/Type/Select/Tree/Index.rst

selectTree
==========

> The selectTree render type is used to display hierarchical data in a

Reviewer: ```suggestion
The :php:`selectTree` render type is used to display hierarchical data in a
```
selectTree is not an english word but a value that you use in TCA, therefore we mark it as PHP code

tree structure.

> A field with the selectTree render type allows you to represent data in

Reviewer: ```suggestion
A field with the :php:`selectTree` render type allows you to represent data in
```

a hierarchical manner, similar to a tree. It is typically used when working

> with database tables that have a hierarchical structure. The field requires
> certain fields to be set in order to function properly. The treeConfig and
> foreign_table fields are mandatory and must be provided to establish the

Reviewer: ```suggestion
with database tables that have a hierarchical structure. The properties  :php:`treeConfig` and
 :php:`foreign_table` are mandatory and must be provided to establish the
```

connection to the relevant database table. Optionally, you can also use

> items or itemsProcFunc to pass the values, but these are not

Reviewer: ```suggestion
:ref:`items <columns-select-properties-items>` or :ref:`itemsProcFunc 
<tca_property_itemsProcFunc>` to pass the values, but these are not
```
And these as well

sufficient on their own. The top-level item in the tree will always represent
the descriptive name of the table.

> Regarding joining several tables, the selectTree field can handle multiple

Reviewer: ```suggestion
Regarding joining several tables, the :php:`selectTree` render type can handle multiple
```

> tables through the configuration options. By appropriately setting up the

Reviewer: The sentece is a little strange and does not really explain how to do that. Either explain how to do that or link to a source or just leave it out?

treeConfig and defining the relationships between the tables, you can create
complex trees that span multiple database tables.

Please note that the selectTree field only handles the presentation of
hierarchical data in a tree structure and does not directly handle the
database operations. The actual data retrieval and manipulation need to

> be implemented separately.

Reviewer: How? do we have a source / example on that?
